.github/workflows/build_test_push.yaml

```
@@ -88,3 +146,22 @@ jobs:
 
       - name: run functional test
         run:  bash functional_test.sh ${{ github.sha }}-server-dev
+  add-git-tags:
+    runs-on: ubuntu-latest
+    needs: functional-tests
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: '0'
+    - name: Compute new tag
+      id: compute_new_tag
+      run: |
+        current_tag=$(git for-each-ref --sort=-v:refname --format '%(refname:lstrip=2)' |  grep -E "^v[0-9]+$" | cut -c 2- | head -n1)
```

nit: might prefer single quote so the `$` is definitely a literal (rather than part of var, though not case here obv)
```suggestion
        current_tag=$(git for-each-ref --sort=-v:refname --format '%(refname:lstrip=2)' |  grep -E '^v[0-9]+$' | cut -c 2- | head -n1)
```